packages/support/src/settings/_variables.color.docs.scss

```diff
@@ -0,0 +1,148 @@
+*/
+
+/*
+Secondary colors
+
+@hide-markup
+
+These are accent colors to provide additional lightness and style to pages looking for a more modern flair. These colors should be used to highlight important features on a page, such as buttons, or for visual style elements, such as illustrations. They should be used sparingly and never draw the eye to more than one piece of information at a time.
+
+Markup:
+<% var colors = [
+  'primary-alt-darkest', 'primary-alt-dark', 'primary-alt', 'primary-alt-light', 'primary-alt-lightest'
+]; -%>
+<div class="">
+    <% colors.forEach(color => { -%>
+      <article class="ds-u-margin-bottom--1">
+        <div class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"></div>
+        <code>$color-<%= color %></code>
+        <code class="c-swatch__label js-swatch-hex ds-u-fill--transparent ds-u-color--gray">#</code>
+      </article>
+    <% }) -%>
+  </div>
+
+
+Style guide: style.color.secondary
+*/
+
+/*
+Background colors
+
+@hide-markup
+
+These colors are used largely for background blocks and large content areas. When alternating between tones, be sure to use enough contrast between adjacent colors.
+
+Markup:
+<% var colors = [
+  'background', 'background-inverse', 'gray-dark', 'gray', 'gray-light', 'gray-lighter', 'gray-lightest'
+]; -%>
+<div class="">
+    <% colors.forEach(color => { -%>
+      <article class="ds-u-margin-bottom--1">
+        <div class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"></div>
+        <code>$color-<%= color %></code>
+        <code class="c-swatch__label js-swatch-hex ds-u-fill--transparent ds-u-color--gray">#</code>
+      </article>
+    <% }) -%>
+  </div>
+
+
+Style guide: style.color.background
+*/
+
+/*
+Status colors
+
+@hide-markup
+
+These colors are used largely to indicate the status of something. Be cautious using these colors to signify something that may contradict what most people would expect the colors to be used for.
+
+Markup:
+<% var colors = [
+  'success', 'success-light', 'success-lighter', 'success-lightest',
+  'warn', 'warn-light', 'warn-lighter', 'warn-lightest',
+  'error-darkest', 'error-dark', 'error', 'error-light', 'error-lighter', 'error-lightest',
+]; -%>
+<div class="">
+    <% colors.forEach(color => { -%>
+      <article class="ds-u-margin-bottom--1">
+        <div class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"></div>
+        <code>$color-<%= color %></code>
+        <code class="c-swatch__label js-swatch-hex ds-u-fill--transparent ds-u-color--gray">#</code>
+      </article>
+    <% }) -%>
+  </div>
+
+
+Style guide: style.color.status
+*/
+
+
+/*
+Additional colors
+
+@hide-markup
+
+Use these colors when the status colors are not specific enough or don't give enough information. In some situations `color-green-lighter` might make more sense to use than `color-success-lighter.`
```

Comment:
What do you think about adding a sentence explicitly saying these colors are identical to others aside from the name?

Reply:
I would even be in favor of removing the duplicate colors from the example and just adding an additional paragraph

Reply:
Something like this:
https://gyazo.com/b3640971f7b674e6719ff11e404bcdff
https://gyazo.com/f712960dc2bc8ec51e8d062c966ca831

I think the advantage of this is that there are 2 other "additional colors" that dont have to do with status colors, currently they are drowned out by the duplicate colors. I also dont think the alternate naming is important enough where it needs its own section, especially when it can be confusing and not obvious the colors are duplicate.

Reply:
I think we can remove them altogether from the color palette page

Reply:
I think its worth surfacing the alternate naming somewhere, like in a guidance section or in text. I think theres no need to have duplicate colors in the examples though.